pallets/roles/src/offences.rs

```diff
@@ -0,0 +1,53 @@
+use super::*;
+use sp_runtime::{Perbill, Saturating};
+use sp_staking::{
+	offence::{Kind, Offence},
+	SessionIndex,
+};
+use tangle_primitives::jobs::ValidatorOffenceType;
+
+#[derive(PartialEq, Clone, Debug, Encode, Decode)]
+pub struct ValidatorOffence<Offender> {
+	/// The current session index in which we report the validators that submitted an invalid
+	/// ethereum log.
+	pub session_index: SessionIndex,
+	/// The size of the validator set in current session/era.
+	pub validator_set_count: u32,
+	/// Authorities that validated the invalid log.
+	pub offenders: Vec<Offender>,
```

Comment:
The comment for this seems unclear, are these offenders or people who validated the offender's offence?

Reply:
No, they are the offenders.. my bad updated the comments